.pre-commit-config.yaml

```diff
@@ -138,9 +140,11 @@ repos:
                 pass_filenames: false
                 verbose: false
       - repo: https://github.com/codespell-project/codespell
-        rev: v2.1.0
+        rev: v2.2.2
         hooks:
               - id: codespell
+                additional_dependencies: [tomli]
```

Comment:
Unfortunately no, because they are different imports and each hook just happens to have decided on using a different one (i.e. pydocstyle uses toml while codespell uses tomli). Both will eventually go away when we drop support for Python < 3.11 since I assume these packages will switch to the stdlib tomllib at that point, but until then we're stuck with this divergence.